build-priv.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# Build opflex and aci-containers private images in non jenkins environment
+# 1. go get github.com/noironetworks/aci-containers
+# 2. mkdir -p $HOME/work && cd $HOME/work
+# 3. git clone https://github.com/noironetworks/opflex opflex-noiro
+# 4. Modify docker/Dockerfile-* and Makefile to reflect DOCKER_USER
+# 5. docker login as DOCKER_USER
+# usage: build.sh <opflex-dir> <aci-containers-dir> <docker-user>
+# example: ./build-priv.sh challa
+
```

Comment:
Good to add `set -euxo pipefail`
https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

Reply:
Thank you for the link. Added that.